libraries/FS/src/FS.cpp

```diff
@@ -187,6 +187,22 @@ File File::openNextFile(const char* mode)
     return _p->openNextFile(mode);
 }
 
+boolean File::seekDir(long position){
+    if(!_p){
+        return false;
+    }
+    return _p->seekDir(position);
+}
+
+boolean File::getNextFileName(String &filename)
+{
+    if (!_p) {
+        return NULL; 
```

Comment:
I see an inconsistency here given that the function `getNextFileName()` shall return a `boolean`.
it is better to `return false` instead.

Reply:
@SuGlider see my comment above. It's better to return `String` directly, not false or NULL.

Reply:
Ok, it make the API easier returning a String as you proposed.

Reply:
i maked the changes, now its return `String` name directly or String empty